internal/transport/http_test.go

```diff
@@ -0,0 +1,141 @@
+// +build go1.16
```

Comment:
This is mainly because http.Client in 1.15 is not able to have a nil TLS config, I do not want to over-complicate the thing because the version EOL, and the alternative is set certs, etc..

Reply:
Yes got it. I will modify the test matrix asap.